var/spack/repos/builtin/packages/r-fpcompare/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RFpcompare(RPackage):
+    """Reliable Comparison of Floating Point Numbers"""
+
+    homepage = "https://github.com/PredictiveEcology/fpCompare"
+    url      = "https://cloud.r-project.org/src/contrib/fpCompare_0.2.3.tar.gz"
+
```

Comment:
Need the `list_url`.

Reply:
Isn't list_url implicit? Because the reasoning for having it is so the archive can be checksummed and we can retrieve the previous version. So using the package that I currently have, I ran "spack checksum r-fpcompare". It returned with all the previous versions and the proper archive URL. For example, this is one of them that came from the checksum cmd:
https://cloud.r-project.org/src/contrib/Archive/fpCompare/fpCompare_0.2.2.tar.gz

And I never specified the archive url.